### ISSUE TITLE
VAE mesh decoder training data

### ISSUE THREAD

**Issue by lan-creator** [microsoft/TRELLIS#236]
Hello, I noticed that you use normal map to calculate color loss as follows:

> a normal map N_m directly derived from the mesh,

However, I try to find the corresponding code here, and I find you didn't use the rendered mesh gt normal map in `structured_latent_vae_mesh_dec.py:line 213`, but instead use normal map from dataloader, where it doesn't provide any normal map data. So, I wonder if you indeed not use normal map recon loss in color loss. Thank you for your reply!
https://github.com/microsoft/TRELLIS/blob/eb83038919f6e1feb63accf3a97a377a608c497d/trellis/trainers/vae/structured_latent_vae_mesh_dec.py#L171-L216
https://github.com/microsoft/TRELLIS/blob/eb83038919f6e1feb63accf3a97a377a608c497d/trellis/datasets/structured_latent2render.py#L90-L112

**Comment by nchatzis** [comment]
Did you find a solution for this? I can also see that `normal_map` is passed as `None` (default) in `training_losses` and the dataset does not provide it anywhere for ground truth.

Update for you: there is a `--save_normal` argument you can pass when calling the blender script inside of `dataset_toolkits/render.py`. Blender throws an error on my custom data but I assume this is how you get GT normal maps.
